### ISSUE TITLE
Prevent pull down to refresh page on Chromium based mobile browsers

### ISSUE THREAD

**Issue by SuperPhatArrow** [monlovesmango/astral#85]
When swiping down to scroll up at the top of the page, Chromium based mobile browsers (e.g. Chrome, Brave) default action is to refresh the entire site. This is not good behaviour for Astral as it has to fetch all new data from relays.

It can be overridden with the following CSS:

```css
html { overscroll-behavior-y: contain; }
```

[Reference](https://developer.chrome.com/blog/overscroll-behavior/#introducing-overscroll-behavior)

**Comment by SuperPhatArrow** [comment]
@monlovesmango I think that inserting this into [MainLayout.vue](https://github.com/monlovesmango/astral/blob/master/src/layouts/MainLayout.vue#L397) would do the trick.

**Comment by monlovesmango** [comment]
thank you for this! I agree. I want to figure out how to capture this action and direct it do to something else, like reload/reset the UI but not completely reload the page. or even load unread feed events. do you have tips around this? if not no worries but if so it would prob help me implement faster.

**Comment by SuperPhatArrow** [comment]
Yes, it must be possible because the Twitter PWA does it. I will look into it, I imagine it will be adding an event listener to the element. I'll report back soon.

**Comment by SuperPhatArrow** [comment]
Ok I have figured it out and put together [an ugly demo](https://superphatarrow.github.io/tests/refresh-override/) for you to try on your phone. 
```css
html, body { overscroll-behavior-y: contain; }
```
I needed to add body to the CSS to cancel the refresh. 

All code is [here](https://github.com/SuperPhatArrow/superphatarrow.github.io/tree/main/tests/refresh-override) including some overly commented JS which will hopefully explain everything! Any questions, please let me know.

**Comment by monlovesmango** [comment]
oh wow thank you so much for this! will def put this in the next major release!!
